flink-connectors/flink-connector-jdbc/src/test/java/org/apache/flink/connector/jdbc/catalog/PostgresCatalogITCase.java

```diff
@@ -154,7 +154,7 @@ public void testArrayTypes() {
         assertEquals(
                 "[+I["
                         + "[1, 2, 3], "
-                        + "[[92, 120, 51, 50], [92, 120, 51, 51], [92, 120, 51, 52]], "
+                        + "[[50], [51], [52]], "
```

Comment:
Why is this change necessary?

Reply:
@snuyanzin Could you elaborate?

Reply:
Since the logic for handling of PG byte[] was changed.
Before if there is an input `{2,3,4}` it translated it to string representation `{\x32, \x33, \x34}` and then translated each symbol to hex `[[92, 120, 51, 50], [92, 120, 51, 51], [92, 120, 51, 52]]`
Now byte array contains hex not of the string representation but of the actual values